app/queries/intent_metrics_query.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+# Query to return the metrics on the intent selections of the students in a
+# specific draw.
+class IntentMetricsQuery
```

Comment:
this is great! we might want to add some more documentation for less-experienced devs, but it's not critical